data/abilities/elastic_hunting/4b283acc-45c0-4de8-b0ac-ac0699e5ab95.yml

```diff
@@ -0,0 +1,33 @@
+---
+- id: 4b283acc-45c0-4de8-b0ac-ac0699e5ab95
+  name: Search for PowerShell ExecutionPolicy Bypass (elastic)
+  description: Search for Sysmon Event 1 powershell records with "ExecutionPolicy" and "Bypass"
+  tactic: hunt
+  technique:
+    attack_id: x
+    name: x
+  platforms:
+    windows:
+      elasticsearch: &cmd

```

Comment:
Typo?

Reply:
The `&cmd` is actually a YAML anchor, since the command and parsers are the same for every platform using them cuts down repeated code.  `&` basically copies everything underneath it in the hierarchy and `*` pastes it into to other keys here.